### ISSUE TITLE
SSL Error

### ISSUE THREAD

**Issue by ogamel** [QISKit/qiskit-core#532]
**Informations**

- **QISKit (Python SDK) version**: 0.5.3
- **Operating system**: OS X 10.13.1 (High Sierra)
- **Python version**: 3.6.3

**What is the current behavior?**
Executing any circuit on the ibmqx5 or ibmqx4 chips yields this error:

```
requests.exceptions.SSLError: HTTPSConnectionPool(host='quantumexperience.ng.bluemix.net', port=443): Max retries exceeded with url: /api/Jobs/5b155fd3619db20042f26ccc?access_token=q2jZBavpo2kXsFKEUpNU92UrADhgBIAGn4p2tED7u9oDsEXaf0aAhIcmwqqm4ETF (Caused by SSLError(SSLError(1, '[SSL: BAD_SIGNATURE] bad signature (_ssl.c:777)'),))'
```

Sometimes there will be a long delay before throwing this error.

This happens whether or not verify=True or False. If False, it first outputs
`-- Ignoring SSL errors.  This is not recommended --`
but it still throws the SSL error

**Steps to reproduce the problem**
```
n=2
qprog = QuantumProgram()
register(Qconfig.APItoken, Qconfig.config['url'],verify=False)
qreg = qprog.create_quantum_register("qr", n)
creg = qprog.create_classical_register("cr", n) # use all n registers
qcirc = qprog.create_circuit("qc", [qreg], [creg])
 ..... # some unitary gates # ...
for i in range(n):
    qcirc.measure(qreg[i],creg[i])
res = qprog.execute("qc", backendname, shots=512, timeout=360)
```

**What is the expected behavior?**
My code was running last week. Then I had an error that seemingly kept my credits at zero. I then upgraded to qiskit 0.5.3 and upgraded to researcher status a few days ago, with 150 credits. Now I get this strange error I have never seen before.

**Suggested solutions**
Make the error messages more informative.
I am unsure of the root cause.

**Comment by diego-plan9** [comment]
Hi @ogamel - can you let us know if you are using a proxy or similar?

**Comment by ogamel** [comment]
Hi @diego-plan9, no proxy. If you have a question on a specific setting I can relay it to my network administrator.

**Comment by diego-plan9** [comment]
Thanks for the quick reply! The reason for checking the existence of a proxy is because we cannot reproduce the issue, and it would be highly likely that a problem with the SSL certificate as a whole would have been detected much earlier and prominently if it affected other users. Can you paste me the output of:

```
import logging
import requests
logging.basicConfig(level=logging.DEBUG)
requests.get("https://quantumexperience.ng.bluemix.net/api/version")
```

on the same environment you are trying to run your snippet of code, and also the result of a command line tool (hopefully `wget` or `curl` are on your system):
```
$ wget https://quantumexperience.ng.bluemix.net/api/version
$ curl -v https://quantumexperience.ng.bluemix.net/api/version
```
Thanks!

**Comment by ogamel** [comment]
Than you Diego. The output of the first part is

```
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): quantumexperience.ng.bluemix.net
DEBUG:urllib3.connectionpool:https://quantumexperience.ng.bluemix.net:443 "GET /api/version HTTP/1.1" 200 None
Out[2]: <Response [200]>
```

the curl command output is rather long, so it is attached in a txt file

[18.06.05 debug.txt](https://github.com/QISKit/qiskit-core/files/2072619/18.06.05.debug.txt)

Thank you for your help

**Comment by diego-plan9** [comment]
Hmm, it seems you are getting an HTTP `200` - meaning no SSL errors at all, which seems puzzling. Can you paste the contents of your `Qconfig.py` file (but please redact and do not paste your `APIToken)?

**Comment by ogamel** [comment]
Speaking to my admins, it seems they were working out some network issues that are now resolved. My code now runs and I don't get the error. 
Since it happened right after I upgraded qiskit version and changed status to researcher, I mistakenly  thought these were the causes. But I suppose, as David Hume would have it, deducing causality by the order of events is a subtle business :)

Thank you for your help Diego.
